nautobot/extras/templates/extras/jobresult.html

```diff
@@ -73,28 +80,35 @@
 
 {% block extra_tab_content %}
     <div role="tabpanel" class="tab-pane" id="json">
-        <div class="row">
-            <div class="col-md-6">
-                <div class="panel panel-default">
-                    <div class="panel-heading">
-                        <strong>Job Inputs</strong>
-                    </div>
-                    <div class="panel-body">
-                        {% include 'extras/inc/json_data.html' with data=result.job_kwargs format="json" %}
+        {% if not result.job_model.job_class.has_sensitive_variables %}
+            <div class="row">
+                <div class="col-md-6">
+                    <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <strong>Job Inputs</strong>
+                        </div>
+                        <div class="panel-body">
+                            {% include 'extras/inc/json_data.html' with data=result.job_kwargs format="json" %}
+                        </div>
                     </div>
                 </div>
-            </div>
-            <div class="col-md-6">
-                <div class="panel panel-default">
-                    <div class="panel-heading">
-                        <strong>Output Data</strong>
-                    </div>
-                    <div class="panel-body">
-                        {% include 'extras/inc/json_data.html' with data=result.data format="json" %}
+                <div class="col-md-6">
+                    <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <strong>Output Data</strong>
+                        </div>
+                        <div class="panel-body">
+                            {% include 'extras/inc/json_data.html' with data=result.data format="json" %}
```

Comment:
I don't think `has_sensitive_variables` prevents a Job from reporting outputs or writing to `result.data`. Nor do I think it should.